query/src/main/scala/filodb/query/exec/rangefn/InstantFunction.scala

```diff
@@ -48,7 +40,7 @@ trait HistToDoubleIFunction extends HistogramInstantFunction {
     * @param value Sample against which the function will be applied
     * @return Calculated value
     */
-  def apply(value: Histogram): Double
+  def apply(value: Histogram, scalarParam: Seq[Double] = Nil): Double
```

Comment:
Why is the scalarParam needed here?

Reply:
and how come it's always double instead of `FuncArg` etc?

Reply:
InstantFunction's need Double parameter. Instead of converting FuncArg to double value in every function I have done it in InstantFunctionMapper.  InstantFunctionMapper apply method converts FuncArgs to ScalarRangeVector based on type of FuncArgs. Then ```getValue(timestamp)``` of ScalarRangeVector is used to get the double value which is passed to the InstantFunction apply